app/api/ltc.py

```diff
@@ -126,6 +126,33 @@ def collapse_rows_new_header_names(df_group, col_map):
     return new_df_subset
 
 
+# cleans up CTP Facility Types and federal/state regulated
+# this is optimized for FL - other states have different labels
+def state_to_ctp(record):
```

Comment:
maybe we can rename to `FL_state_to_ctp` since it's a FL specific function?

Reply:
good idea!